MaterialProgressBar/src/main/ets/components/basic/CircularPath.ets

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2022 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import ProgressBar from './ProgressOfTheBar';
+
+export default class CircularPath implements ProgressBar {
+  private startPoint: number[];
+  private endPoint: number[] = [0, 0];
+  private angle: number = 0;
+  private offsetAngle: number = 0;
+  private radius: number = (100);
+  private strokeWidth: number = (5);
+  private pathCmd: string = '';
+  progress: number = 0;
+  color: Color;
+  maxProgress: number = 100;
+  private isReset: boolean = true;
+
+  setAngle(angle: number): CircularPath{
+    this.offsetAngle = angle;
+    return this;
+  }
+
+  getAngle(): number{
+    return this.offsetAngle;
+  }
+
+  setColor(color: Color): CircularPath{
+    this.color = color;
+    return this;
+  }
+
+  getColor(): Color{
+    return this.color;
+  }
+
+  public setProgress(progress: number): CircularPath{
+    this.progress = Math.max(0, Math.min(this.maxProgress, progress));
+    this.update();
+    return this;
+  }
+
+  public getProgress(): number{
+    return this.progress;
+  }
+
+  setMaxProgress(max: number): CircularPath{
+    this.maxProgress = max;
+    return this;
+  }
+
+  getMaxProgress(): number{
+    return this.maxProgress;
+  }
+
+  public setRadius(radius: number): CircularPath{
+    //    this.radius = vp2px(radius);
+    this.radius = (radius);
+    return this;
+  }
+
+  public setStrokeWidth(strokeWidth: number): CircularPath{
+    this.strokeWidth = (strokeWidth);
+    //    this.strokeWidth = vp2px(strokeWidth);
```

Comment:
Remove the commented code. Applies to all the files. Please check & handle

Reply:
Done